bootstrap/Dockerfile

```diff
@@ -1,23 +1,17 @@
 # bootstrapper-builder contains the tool chain we need to build the image.
 # This primarily means glide.
-FROM gcr.io/kubeflow-images-public/bootstrapper-builder:v20180712-4adef8c0 as builder
+ARG GOLANG_VERSION=1.10.3
+ARG BUILDER_IMG=gcr.io/kubeflow-images-public/bootstrapper-builder
```

Comment:
Nit: Why are you taking the image tag and image path as separate arguments? Why not just take a single image which is the full path of the image?

Reply:
for `make push-builder-latest` it's easier to keep the builder image separate from its tag.
```
push-builder-latest: push-builder
        gcloud container images add-tag --quiet $(BUILDER_IMG):$(TAG) $(BUILDER_IMG):latest --verbosity=info
        @echo created $(BUILDER_IMG):latest
```

Similar logic is used with IMG and TAG.